.lando.yml

```diff
@@ -21,6 +21,16 @@ services:
 #    overrides:
 #      ports:
 #        - "32654:32654"
+  node:
+    type: node:16
+    build_as_root:
+      - apt update -y && apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb -y
+    build:
+      - cd testing && npm install && npm install -g cypress@12.3.0
+    overrides:
+      environment:
+        LOCAL: "LOCAL"
+        LANDO_URL: "http://vote-gov.lndo.site/"
   appserver:
```

Comment:
Use the node version 18 that is already commented out above and apply these settings to it.

Reply:
updated format to use verson 18